pom.xml

```diff
@@ -16,6 +16,7 @@
 	<properties>
 		<enforceFailOnUIDependencyInCore>true</enforceFailOnUIDependencyInCore>
  		<tycho.scmUrl>scm:git:https://github.com/jbosstools/jbosstools-base.git</tycho.scmUrl>
+ 		<sonar.test.inclusions>**/*Test.*,**/test/**/*</sonar.test.inclusions>
```

Comment:
According to https://github.com/jbosstools/jbosstools-build/blob/master/parent/pom.xml#L290 the list is slightly different